src/tests/bulkConcepts/addBulkConcepts.test.jsx

```diff
@@ -214,7 +216,7 @@ describe('Add Bulk Concepts', () => {
     const other = wrapper.find('#otherSourcesOption');
     other.simulate('click');
     const input = wrapper.find('#sourceSearch');
-    input.simulate('change', { target: { value: '' } });
+    input.simulate('change', { target: { value: 'testSource' } });
```

Comment:
How is this related to execute a search only when ENTER is pressed?

Reply:
Because to do a search one is supposed to first press enter hence I first had to simulate change with the value I would like to search for.